### PR TITLE
Docs conversion!

### PR DESCRIPTION
**Pull request by dbanksdesign**
*Issue #, if available:*

*Description of changes:* LOTS of changes. Mostly removing old amplify docs code and rolling our own docs site. 

https://docs-conversion-2.d3inl0muob87le.amplifyapp.com/

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ericclemmons — APPROVED** [review]
Just reviewed https://docs-conversion-2.d3inl0muob87le.amplifyapp.com/components/authenticator and giving a 👍  to incrementally make improvements to _that_ version.

File: docs/src/components/Layout/SecondaryNav.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import { Heading } from '@aws-amplify/ui-react';
+import { useRouter } from 'next/router';
+
+const NavLink = ({ href, children }) => {
+  const { pathname } = useRouter();
+  const isCurrent = pathname === href;
+
+  return (
+    <Link href={href}>
+      <a className={`docs-secondary-nav-link ${isCurrent ? 'current' : ''}`}>
```
Comment (reesscot) [inline docs/src/components/Layout/SecondaryNav.tsx:11]: Why doesn't this result in nested `a` tags?

File: docs/src/components/Layout/index.tsx [hunk 2]
```diff
@@ -104,179 +40,43 @@ export default function Page({
...
-    </>
+
+      <Alert
+        variation="warning"
+        isDismissible={true}
+        position="fixed"
+        bottom={`${tokens.space.large}`}
+        left={`${tokens.space.large}`}
+        right={`${tokens.space.large}`}
+        style={{ zIndex: '9999' }}
```
Comment (reesscot) [inline docs/src/components/Layout/index.tsx:74]: Should we add this to our base styles?

**Review by reesscot — APPROVED** [review]
LGTM!

File: docs/src/components/Layout/SecondaryNav.tsx (hunk 1 above)
Comment (dbanksdesign) [reply docs/src/components/Layout/SecondaryNav.tsx:11]: The way Next's Link works: https://nextjs.org/docs/api-reference/next/link

**Pull request merged**